src/Utils.ts

```diff
@@ -60,3 +60,17 @@ export function castBreakpointsToIntegers(breakpoints: {
     {}
   )
 }
+
+/**
+ * Use this function to memoize any function
+ */
+export function memoize<F extends (...args: any[]) => void>(func: F) {
```

Comment:
I just found a random memoize function on some article and typed it as best as I could.. any reason why I shouldn't import `lodash` or a third-party unit-tested memoize package?

Reply:
Copying this in is ideal 👍

Reply:
You should probably use https://github.com/alexreardon/memoize-one here, which is much more performant.